src/slf4j_timbre/timbre_service_provider.clj

```diff
@@ -0,0 +1,40 @@
+(ns slf4j-timbre.timbre-service-provider
+  (:import (com.github.fzakaria.slf4j.timbre TimbreLoggerFactory)
+           (org.slf4j ILoggerFactory IMarkerFactory)
+           (org.slf4j.helpers BasicMDCAdapter BasicMarkerFactory)
+           (org.slf4j.spi MDCAdapter))
+  (:gen-class :name com.github.fzakaria.slf4j.timbre.TimbreServiceProvider
+              :implements [org.slf4j.spi.SLF4JServiceProvider]
+              #_#_:methods [[getLoggerFactory [] org.slf4j.ILoggerFactory]
+                        [getMarkerFactory [] org.slf4j.IMarkerFactory]
+                        [getMDCAdapter [] org.slf4j.spi.MDCAdapter]
+                        [getRequestedApiVersion [] String]
+                        [initialize [] void]]
+              )
+  )
```

Comment:
This should probably be removed since it's all commented out.

Reply:
Yeah, right.